packetery/template/checkout/checkout_script.latte

```diff
@@ -114,7 +114,15 @@
 			}, widgetOptions);
 		});
 
-		$(document).on('change', '#payment input[type="radio"]', function () {
+		var currentPaymentMethod = $('#payment input[type="radio"]:checked').val();
+		$(document).on('change', '#payment input[type="radio"]:checked', function (e) {
```

Comment:
Je spolehlivé věšet posluchač jen na :checked inputy a automaticky předpokládat, že target je po změně :checked?
Já nevím, takže bych věšel na všechny a kontroloval bych, jestli je target checked.

Reply:
On je to delegovaný listener a máme to otestované i od eshopu.